src/bicep/add-ons/policy-guardrails/deploy-policy-intiatives.ps1

```diff
@@ -0,0 +1,157 @@
+#[CmdletBinding()]
+param (
+    [string]$RootFolderPath = 'C:\Users\brsteel\OneDrive - Microsoft\Documents\policies',
```

Comment:
I would remove the file path for your local file system and would create a comment-based help section at the top of the script using the following reference to create examples: https://learn.microsoft.com/en-us/powershell/scripting/developer/help/comment-based-help-keywords?view=powershell-7.5